generative_data_prep/__main__.py

```diff
@@ -132,6 +132,13 @@ def add_pipeline_args(subparser: argparse.ArgumentParser):
         help="The number of CPU workers to multi-process run tokenization over, if the previous run failed you need to \
         decrease this number.",  # noqa: E251
     )
+    subparser.add_argument(
+        "--do_not_delete_split_jsonls",
```

Comment:
super nit: Wanna call this variable `keep_split_jsonls`. Slightly easier to read.